### DISCUSSION TITLE
inlock sim with at&t

### DISCUSSION THREAD

**Discussion by amerelfallah** [community/community#55257] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

I want fix phone 12 pro pls inlock sim with at&t

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @amerelfallah , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist or your phone provider for assistance. They should be able to help you with these applications!
